css/collection-card.styl

```diff
@@ -1,26 +1,53 @@
 .collections-card-list
   @extends $owned-card-list
-  
+
 .collection-card
   @extends $owned-card
   background: url('/assets/simple-pattern.png') center center repeat
   border-radius: 1em
   position: relative
+  height: 270px
 
-  .name
-    margin-bottom: 1em
-    height: 2.9em
-    overflow: hidden
-    text-overflow: ellipsis
-    line-height: 1.45em
-    max-height: 2.9em
+  .details
+    padding: 1em
+    text-align: left
+    height: 80px
+    transition: height 0.5s ease
+    position: absolute
+    bottom: 0
+    left: 0
+    right: 0
+    justify-content: space-between
```

Comment:
This appears to largely be a copy/paste of the CSS from project cards. If they're going to be shared, then let's get the shared bits into the owned card styles in common.styl and the divergent styles would go in either collection-card.styl and project-card.styl respectively. That being said, there's some CSS in the project cards that doesn't make sense anymore. All of the flex stuff on the details container for instance. We should be removing cruft styles as the styles change.